src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,7 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => <PostInfo key={post.id} post={post} />)}
```

Comment:
You should handle the case where 'posts' array is empty or null. Add a condition to render a message or a component when 'posts' is empty or null.